README.md

```diff
@@ -54,19 +53,14 @@ const options = {
     }
 };
 
-server.register({
-    register: require('good'),
+await server.register({
+    plugin: require('good'),
     options,
-}, (err) => {
+});
 
-    if (err) {
-        return console.error(err);
-    }
-    server.start(() => {
-        console.info(`Server started at ${ server.info.uri }`);
-    });
+await server.start();
 
-});
+console.info(`Server started at ${ server.info.uri }`);
```

Comment:
We should add something about the supported versions of hapi somewhere in the readme.

Reply:
What format do you want this in. I looked at all the other plugins in the hapi namespace and didn't see an others doing this as was just going to follow whatever format is there.

Reply:
@arb Just wanted to check if you had a chance to think about this. Don't mean to rush know we all get busy just excited ;)

Reply:
Put it high in the README. Doesn't matter to me what the format is, just so long as it's there so when people open issues complaining, it is documented.

Reply:
@arb Done!